src/twisted/web/http.py

```diff
@@ -420,8 +426,8 @@ def datetimeToLogString(msSinceEpoch=None):
     @rtype: C{str}
     """
     if msSinceEpoch == None:
-        msSinceEpoch = time.time()
-    year, month, day, hh, mm, ss, wd, y, z = time.gmtime(msSinceEpoch)
+        msSinceEpoch = time()
```

Comment:
A `# pragma: no cover` seems reasonable here as `msSinceEpoch` is for testing anyway.

Reply:
It's actually always passed in both tests and real code.